torchvision/prototype/models/vision_transformer.py

```diff
@@ -443,3 +380,74 @@ def vit_l_32(*, weights: Optional[ViT_L_32_Weights] = None, progress: bool = Tru
         progress=progress,
         **kwargs,
     )
+
+
+def interpolate_embeddings(image_size: int, patch_size: int, model_state: Mapping[str, torch.Tensor], reset_heads=False):
```

Comment:
The `model_state` should be of type `'OrderedDict[str, Tensor]'` as you can see [here](https://github.com/malfet/pytorch/blob/master/torch/nn/modules/module.py#L1376-L1377).

Also, could you add the types for other missing vars (`reset_heads`, the return type etc).